src/pages/Story.js

```diff
@@ -109,76 +109,113 @@ const Story = (props) => {
+                    <img
+                      className='avatar'
+                      src={`https://avatars.dicebear.com/api/jdenticon/${story.author.username}.svg`}
+                      alt='Default User Avatar'
+                    ></img>
+                  </div>
+                  <div className='story-buttons-container-top'>
+                    {editMode && !editor ? (
+                      <>
+                        <Button
+                          className='btn btn-default'
+                          data-cy='btn-edit'
+                          onClick={() => setEditor(true)}
+                        >
+                          Edit
+                        </Button>
+                      </>
+                    ) : (
+                      ''
+                    )}
+                    {
+                      <>
+                        <Button className='share-story' onClick={togglePopup}>
+                          <i className='eos-icons'> share </i>
+                        </Button>
+                        <Button className='share-story' onClick={copy}>
+                          <i className='eos-icons'>content_copy</i>
+                        </Button>
+                      </>
+                    }
                   </div>
                 </div>
               </div>
 
-              {editor ? (
-                <div data-cy='edit-description'>
-                  <MarkdownEditor
-                    callback={(html) => {
-                      setDescription(html)
-                    }}
-                  />
-                </div>
-              ) : (
-                <div
-                  className='story-description'
-                  dangerouslySetInnerHTML={{ __html: story.Description }}
-                  data-cy='story-description'
-                />
-              )}
-              <div className='story-buttons-container'>
-                {editMode && !editor ? (
-                  <>
-                    <Button
-                      className='btn btn-default'
-                      data-cy='btn-edit'
-                      onClick={() => setEditor(true)}
-                    >
-                      Edit
-                    </Button>
-                  </>
+              <div className='flex flex-row'>
+                {editor ? (
+                  <div data-cy='edit-description' className='story-editor'>
+                    <MarkdownEditor
+                      callback={(html) => {
+                        setDescription(html)
+                      }}
+                    />
+                  </div>
                 ) : (
-                  ''
+                  <div className='flex flex-row'>
+                    {story.Attachment.length !== 0 ? (
+                      <div className='gallery-container flex-column'>
+                        <Gallery imageArray={story.Attachment} />
+                      </div>
+                    ) : (
+                      ''
```

Comment:
we dont need the ternary expression if we are not returning anything in second condition.
simply do

```{!!story.Attachment.length && (
          <div className='gallery-container flex-column'>
            <Gallery imageArray={story.Attachment} />
          </div>)
}
```